doc/getting-started/apl-nuc.rst

```diff
@@ -62,6 +62,12 @@ complete this setup.
 
       # swupd autoupdate --disable
 
+   .. note::
+
+      Clear Linux will automatically grab the latest version available during the
+      installation process and put it on your system. If you wish to use a specific
+      version (such as 25130), you can enforce that using ``swupd verify --fix --picky -m 25130``
```

Comment:
Your choice of saying "enforce" makes me wonder, will this command **prevent** an update to a newer version, or just **restore** the system to the specified version if it (accidentally) gets changed to a newer one?

Reply:
It will restore the system to that specific version. The Clear Linux installer **will** pull down the latest version of Clear and install that one (it's not accidental), this is how you can force your system to go back to the version we specify.

Reply:
I have changed the phrasing, can you let me know if you like it better?

Adding new bundles to your system does not update it, it installs the bundle for the current version.

Restoring Clear Linux to an earlier version will preserver all bundles that are installed. What I don't know is what happens if a bundle did not exist in the version you're restoring it to.